app/models/facility.rb

```diff
@@ -1,3 +1,3 @@
 class Facility < ActiveRecord::Base
-    has_many :bill
+    has_many :bills
```

Comment:
[Style]
Please fix the spacing here so that the indentation is 2 spaces, not four.